doc/nrf/releases_and_maturity/releases/release-notes-2.5.2.rst

```diff
@@ -0,0 +1,117 @@
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.5.2**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.5.2`_.
+
+IDE and tool support
+********************
+
+`nRF Connect for Visual Studio Code extension <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.5.2.
+
+:ref:`gs_app_tcm`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.5.2_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.5.2_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.5.2`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_252_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+Bluetooth
+---------
+
+* Fixed an issue that affects SoftDevice Controller when running in continuous scan mode.
+  The controller may end up in an irresponsive state due to this issue.
+  See the :ref:`changelog <nrfxlib:softdevice_controller_changelog>` of the SoftDevice Controller library for additional information.
+
+Matter
+------
+
+* Fixed:
+
+  * An issue where the Matter device's application crashes if the DNS resolve response is processed with the TXT record's data size equal to ``0`` (KRKNWK-18256).
+  * An issue with the IPv6 address update in the DNS server for Matter over Wi-Fi (KRKNWK-18256).
+  * An issue with a memory leak in the deferred attribute persister (KRKNWK-18221).
+
+Wi-Fi
+-----
+
+* Added support for band selection in shell and API.
+
+* Fixed:
+
+  * An issue with QSPI initialization on a custom board with nRF5340 and nRF7002 (SHEL-2372).
```

Comment:
This was a known issue in 2.5.1, but missed adding to the known issues list, so, may be we can add this now with 2.5.1 tag.